src/main_loop.rs

```diff
@@ -94,7 +96,9 @@ impl MainLoop {
         let session_config = self.session_config.clone();
         let cache = self.spotifyd_state.cache.clone();
 
-        Session::connect(session_config, creds, cache).await
+        Session::connect(session_config, creds, cache, false)
+            .await
+            .map(|(session, _creds)| session)
```

Comment:
```suggestion
            .0
```

Should also work, or am I missing something?

Reply:
Yes, because we're mapping a result, so we can't just access the first field of the tuple directly. Using `?` then requires us to wrap the expression in `Ok`. I think this is the most readable.